### PR TITLE
Write documentation

### PR DESCRIPTION
**Pull request by StackOverflowExcept1on**
Preview README: https://github.com/gear-foundation/gstd-fluent/tree/documentation
Docs: https://dapp.rs/gstd-fluent/gstd_fluent

### CODE REVIEW

**Review by shamilsan — APPROVED** [review]
Awesome ❤️‍🔥

File: src/lib.rs [hunk 1]
```diff
@@ -1,3 +1,147 @@
...
+//!         .execute()
+//!         // ^ when you call `execute()` it calls the desired gstd function
+//!         .expect("Unable to reply");
+//!
+//!     // for example, you want to specify an explicit gas limit and value:
+//!     builder::send(msg::source(), payload.clone())
+//!         .with_delay(30)
+//!         // add two new lines:
+//!         .with_gas_limit(1_000_000)
+//!         .with_value(1_000) //value=0 by default
```
Comment (shamilsan) [inline src/lib.rs]: Whitespace after `//` as in other comments.
```suggestion
//!         .with_value(1_000) // value=0 by default
```

File: src/lib.rs [hunk 2]
```diff
@@ -1,3 +1,147 @@
...
+//!
+//!     // instead of replacing `msg::send_delayed` with `msg::send_with_gas_delayed`
+//!     // you can just add two newlines
+//! }
+//! ```
+//!
+//! ### Builder syntax
+//!
+//! ```ignore
+//! # const _: &'static str = stringify! {
```
Comment (breathx) [inline src/lib.rs:55]: What is this

File: src/lib.rs [hunk 3]
```diff
@@ -1,3 +1,147 @@
...
+//! #![no_std]
+//! # };
+//!
+//! use gstd_fluent::{
+//!     self as builder,
+//!     gstd::{self, prelude::*, ActorId, CodeId, ReservationId},
+//! };
+//!
+//! // this is pseudo-code to show all possible methods
```
Comment (breathx) [inline src/lib.rs]: All comment from capital letter please

**Review by breathx — APPROVED** [review]

File: src/lib.rs (hunk 2 above)
Comment (StackOverflowExcept1on) [reply src/lib.rs:55]: This is a hack to include `std::` in doctest. I took this from gstd source code. Otherwise it won't compile.

**Pull request merged**